activities/service/v1/notifications.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+package io.spoud.sdm.activities.v1;
+
+option java_multiple_files = true;
+option java_package = "io.spoud.sdm.activities.v1";
+option go_package = "activities.v1";
+
+import "google/protobuf/empty.proto";
+import "activities/domain/v1/domain.proto";
+
+service notificationsService {
+  rpc updateNotificationChannelsConfiguration (UpdateNotificationChannelConfiguration) returns (google.protobuf.Empty);
+  rpc getNotificationChannelsConfiguration (google.protobuf.Empty) returns (NotificationChannelsConfigurationResponse);
+}
+
+message UpdateNotificationChannelConfiguration{
+  NotificationConfiguration configuration = 1;
+}
+message WebhookConfiguration{
+  string webhook_url = 1;
+  map<string, string> webhook_headers = 2;
+}
+
+message EmailConfiguaration{
+  string email = 1;
+}
+
+message NotificationChannelsConfigurationResponse{
+  repeated NotificationConfiguration configurations = 1;
+}
+
+message NotificationConfiguration{
+  string id = 1;
+  string label = 2;
+  repeated io.spoud.sdm.activities.domain.v1.ActivityType.Type notify_only_on_activity_type = 3;
```

Comment:
What about filters? Because right now you have the activity type but we could imagine filtering by path (like we discussed).

Look at how we filter other entities for example in search. I think it would be nice and dynamic to have that here too

Reply:
add those filters as a list since you could want to receive by more than one path or entity

Reply:
I would suggest to have a message ActivityFilter and to have it as repeated.

In here: 
```
repeated ActivityFilter activity_filters = 3;
```

Bellow
```
message ActivityFilter{
   activity_type = 1;
   path = 2;
   entity_type = 3;
}
```